source/admin_manual/list_of_events.rst

```diff
@@ -2333,6 +2333,38 @@ happens.
 | object_id             | Object ID in the storage                             |
 +-----------------------+------------------------------------------------------+
 
+fs_dictmap_max_bucket_changed
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. versionadded:: 2.3.13
+
+This event is sent whenever the ``max_bucket`` value for a mailbox changes.
+There can be two situations this happens. Either a new mail is added to a
+mailbox, where the current bucket is found to be filled and a next bucket is
+started to get filled (``reason == file``).
+
+Besides the latter, expected situation, Dovecot emits this event if it
```

Comment:
latter=second of two choices, but you're referring to the first choice. I'm not sure if it's correct to say "former" here or not. Maybe just avoid it? "Besides the expected situation, ..."

Reply:
Yes thank you, sounded correct somehow in the first place.